lib/Gocdb_Services/Config.php

```diff
@@ -19,26 +21,33 @@
  *
  * @author David Meredith <david.meredith@stfc.ac.uk>
  * @author John Casson
+ *
+ * @SuppressWarnings(PHPMD.CamelCaseMethodName)
+ * @phpcs:disable PSR1.Methods.CamelCapsMethodName.NotCamelCaps
```

Comment:
what is the effect of this?

Reply:
Suppressing
![image](https://user-images.githubusercontent.com/20706670/204580837-6080dc24-5c75-450b-8c0c-6a96db84e9e2.png)
which would need wider changes.